README.md

```diff
@@ -111,35 +113,45 @@ Go to the [Push Credentials page](https://www.twilio.com/console/voice/credentia
 
 Once the app is ready for store submission, update the plist with “APS Environment: production” and create another Push Credential with the same VoIP Certificate but without checking the sandbox option.
 
-<img src="Images/add-push-credential.png"/>
+<kbd><img src="Images/add-push-credential.png"/></kbd>
 
 Now let's go back to `server.py` and update the Push Credential SID. The Push Credential SID will now be embedded in your access token.
 
     PUSH_CREDENTIAL_SID = 'CR***'
 
 ### <a name="bullet9"></a>9. Configure Xcode project settings for push notifications
-On the project’s Capabilities tab, enable “**Push Notifications**”, and enable both “**Voice over IP**” and “**Audio, AirPlay and Picture in Picture**” capabilities in the Background Modes
+On the project’s Capabilities tab, enable “**Push Notifications**”.
+In Xcode 8 or earlier, enable both “**Voice over IP**” and “**Audio, AirPlay and Picture in Picture**” capabilities in the Background Modes
+
+<kbd><img src="Images/xcode-project-capabilities.png"/></kbd>
 
-<img src="Images/xcode-project-capabilities.png"/>
+In Xcode 9+, add a "**UIBackgroundModes**" dictionary with "**audio**" and "**voip**" to the app's plist.
+```
+<key>UIBackgroundModes</key>
+<array>
+  <string>audio</string>
+  <string>voip</string>
+</array>
+```
```

Comment:
This makes sense, since the GUI pictured above no longer lets you select `voip` in Xcode 9. 👍